crates/rspack_loader_swc/src/compiler.rs

```diff
@@ -610,3 +642,25 @@ impl IntoSwcComments for SingleThreadedComments {
     }
   }
 }
+
+fn get_swc_core_version() -> String {
+  let workspace_toml = cargo_toml::Manifest::from_str(CARGO_TOML)
```

Comment:
I have a small question to ask: Would the `cargo_toml` parsing here be optimized into a constant during compilation?

Reply:
I think this can be achieved by using compile time environment variables: https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-env

Reply:
This is a good question. Lemme fix this ;-)